src/main/java/org/opensearch/ad/NodeStateManager.java

```diff
@@ -286,12 +286,21 @@ public void addPressure(String nodeId, String detectorId) {
     public void resetBackpressureCounter(String nodeId, String detectorId) {
         Map<String, BackPressureRouting> routingMap = backpressureMuter.get(detectorId);
         if (routingMap == null || routingMap.isEmpty()) {
+            // TODO: confirm with Kaituo, do we need to remove detector's backpressure counter for all nodes/
```

Comment:
@kaituo , can you confirm if it's ok to remove detector's backpressure counter for all nodes here? Or we just need to remove counter for the passed in node?